apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/ConnectionPoolOptions.cs

```diff
@@ -59,8 +59,11 @@ public LogLevel LogLevel
                 {
                     return (LogLevel) underlyingLevel;
                 }
-                // ReSharper disable once NotResolvedInText
-                throw new ArgumentOutOfRangeException("LogLevel");
+                else
+                {
+                    // ReSharper disable once NotResolvedInText
+                    throw new ArgumentOutOfRangeException("LogLevel");
+                }
```

Comment:
I don't think you should throw from a getter, but if it is necessary, then you might want to use `GaxPreconditions.CheckEnumValue` if possible.